templates/cli/lib/commands/generic.js.twig

```diff
@@ -3,11 +3,50 @@ const { Command } = require("commander");
 const Client = require("../client");
 const { sdkForConsole } = require("../sdks");
 const { globalConfig, localConfig } = require("../config");
-const { actionRunner, success, parseBool, commandDescriptions, log, parse } = require("../parser");
+const { actionRunner, success, parseBool, commandDescriptions, error, parse, drawTable } = require("../parser");
 {% if sdk.test != "true" %}
 const { questionsLogin, questionsListFactors, questionsMfaChallenge } = require("../questions");
 const { accountUpdateMfaChallenge, accountCreateMfaChallenge, accountGet, accountCreateEmailPasswordSession, accountDeleteSession } = require("./account");
 
+const whoami = new Command("whoami")
+    .description(commandDescriptions['whoami'])
+    .configureHelp({
+        helpWidth: process.stdout.columns || 80
+    })
+    .action(actionRunner(async () => {
+        if (globalConfig.getEndpoint() === '' || globalConfig.getCookie() === '') {
+            error("No user is signed in. Run appwrite login. ..... ");
+            return;
+        }
+
+        let client = await sdkForConsole(false);
+
+        let account;
+
+        try {
+            account = await accountGet({
+                sdk: client,
+                parseOutput: false
+            });
+        } catch (error) {
+            error("No user is signed in");
+            return;
+        }
+
+        success("Signed in");
```

Comment:
```suggestion
```
I dont think this log is needed, just showing data sounds better to me for easier parsing